tasmota/tasmota_xsns_sensor/xsns_63_aht1x.ino

```diff
@@ -56,12 +59,15 @@
 #define AHT_TEMPERATURE_OFFSET  50
 #define KILOBYTE_CONST          1048576.0f
 
-#define AHT1X_CMD_DELAY      40
-#define AHT1X_RST_DELAY      30
+#define AHT1X_CMD_DELAY      20
+#define AHT1X_RST_DELAY      10
 #define AHT1X_MEAS_DELAY     80 // over 75ms in datasheet
 
-#ifdef USE_AHT2x
-  #define AHTX_CMD     0xB1 // Cmd for AHT2x
+#ifdef USE_AHT3x
```

Comment:
I'm not a big fan of having a compile type option to make it AHT20 or AHT30 compatible but never both. Can you detect the type of device by probing an internal value, and adapt accordingly?

Reply:
Maybe probing with the different CMD hex values?

Reply:
Technically, there's no need to differentiate between AHT20 and AHT30 during compilation, as test results suggest they are fully compatible. If the command code is corrected and a delay is implemented after sending the start measurement command, both AHT20 and AHT30 will operate correctly. I couldn’t find a way to programmatically distinguish between them. The addition of the AHT30 driver selection was only made to ensure the corresponding name is displayed in the Tasmota interface.

Reply:
Ok I understand and I think we can merge as-is. However I would largely prefer a common code for ATH10/20/30 and dynamic selection at run-time. This is in the philosophy of Tasmota.
Could you propose a future evolution in that direction?